core/ui/PageTemplate.tid

```diff
@@ -14,7 +14,7 @@ tc-page-container tc-page-view-$(storyviewTitle)$ tc-language-$(languageTitle)$
 
 <$set name="storyviewTitle" value={{$:/view}}>
 
-<$set name="tv-storyview-single-tiddler-mode" value={{{ [<storyviewTitle>getstoryviewsingletiddlermode[]] }}}>
+<$set name="tv-storyview-single-tiddler-mode" value={{{ [<storyviewTitle>getstoryviewmode[singletiddlermode]] }}}>
```

Comment:
I'm not sure, but I think the history-tab caption needs to be changed too?

Reply:
Hi @pmario the open/history tab caption is handled here:

https://github.com/Jermolene/TiddlyWiki5/blob/single-tiddler-mode/core/ui/SideBar/Open.tid#L3

Reply:
ok